### PR TITLE
Patch 81

### PR DESCRIPTION
**Pull request by elnur004**

### CODE REVIEW

File: src/files.js [hunk 1]
```diff
@@ -70,3 +70,51 @@ export default {
...
+import cup3_4 from '../cup3_4'
+import dash from '../dash'
+import gallon1 from '../gallon1'
+import pinch from '../pinch'
+import pint1 from '../pint1'
+import quart1 from '../quart1'
+import tablespoon_1 from '../tablespoon_1'
+import teaspoon_1 from '../teaspoon_1
+
+export default {
```
Comment (vadim9999) [inline src/files.js]: please add all this variables into block export default that located in up. export default might be one in js and defines in the end

File: src/files.js [hunk 2]
```diff
@@ -104,6 +104,7 @@ export default {
   ultimateGroceryList,
   groceryListWithUserRelations,
   dbIngredients,
+  cooking_times,
```
Comment (vadim9999) [inline src/files.js]: please delete it, because we haven't this variable

**Comment by elnur004** [comment]
only cooking_times doesn't exist. delete all these 4 variables?
cause i don't know upper 3 variables exist or not

**Comment by vadim9999** [comment]
don't delete this 3 files, because it's is in import

**Pull request merged**